.github/workflows/end-to-end.yml

```diff
@@ -0,0 +1,24 @@
+name: Tests: Cypress e2e 
```

Comment:
Sorry, can you also update the file name as well?

Reply:
@ermish just making sure you want to use the toolname for the filename?